modules/statics/src/coins.ts

```diff
@@ -953,6 +955,16 @@ export const coins = CoinMap.fromCoins([
     BaseUnit.BSC,
     [...ETH_FEATURES_WITH_MMI, CoinFeature.TSS, CoinFeature.TSS_COLD, CoinFeature.EVM_WALLET]
   ),
+  account(
+    'd0d44124-c7e9-4214-96ae-fbc6856ee3c2',
+    'bnb',
+    'BNB Smart Chain',
+    Networks.main.bnb,
+    18,
+    UnderlyingAsset.BNB,
+    BaseUnit.BNB,
+    [...ETH_FEATURES_WITH_MMI, CoinFeature.TSS, CoinFeature.TSS_COLD, CoinFeature.EVM_WALLET]
+  ),
```

Comment:
Should we use a unique uuidv4 for these "copied" tokens? Or should they get their own uuidv4

Reply:
nevermind it forces a different uuid